examples/snippets/modules/ROOT/examples/build-akka-streamlets-scala/step3/src/test/scala/com/example/TestProcessorSpec.scala

```diff
@@ -0,0 +1,52 @@
+package com.example
+
+//tag::imports[]
+import akka.actor._
+import akka.stream._
+import akka.stream.scaladsl._
+import akka.testkit._
+
+import org.scalatest._
+import org.scalatest.concurrent._
+
+import cloudflow.streamlets._
+import cloudflow.streamlets.avro._
+import cloudflow.akkastream._
+import cloudflow.akkastream.scaladsl._
+import cloudflow.akkastream.testkit._
+import cloudflow.akkastream.testkit.scaladsl._
+//end::imports[]
+
+//tag::test[]
+class TestProcessorSpec extends WordSpec with MustMatchers with BeforeAndAfterAll {
+
+  private implicit val system = ActorSystem("AkkaStreamletSpec")
+  private implicit val mat = ActorMaterializer()
+
+  //tag::afterAll[]
+  override def afterAll: Unit = {
+    TestKit.shutdownActorSystem(system)
+  }
+  //end::afterAll[]
+
+  "An TestProcessor" should {
```

Comment:
[super-duper minor]: 
```suggestion
  "A TestProcessor" should {
```